### PR TITLE
SRL-279: installed module and exposed non-taxonomy fields

### PR DESCRIPTION
**Pull request by melwong-jcc**

### CODE REVIEW

File: config/config-default/core.extension.yml [hunk 1]
```diff
@@ -105,6 +106,7 @@ module:
   search: 0
   search_api: 0
   search_api_db: 0
+  sendgrid_integration: 0
```
Comment (robdecker) [inline config/config-default/core.extension.yml]: This line should be removed since `sendgrid` is handled by `config_split`

**Review by robdecker — CHANGES_REQUESTED** [review]

File: config/config-default/core.extension.yml (hunk 1 above)
Comment (melwong-jcc) [reply config/config-default/core.extension.yml]: Good catch and a missed merge correction.

Comment (melwong-jcc) [reply config/config-default/core.extension.yml]: Corrected core extension. If all looks good, just need to deal with composer.lock and field_counties conflict.

**Comment by melwong-jcc** [comment]
@robdecker Ah. You guys create branches off develop which I also overlooked in the README. I noticed that field.field.node.court.field_counties.yml is missing compared to master. Going to delete field.field.node.court.field_counties.yml from my branch then.

**Comment by melwong-jcc** [comment]
Cancelling pull request test failed plus better to start over on a clean develop branch.

**Pull request closed without merging**